sharding/test/callbacks_test.go

```diff
@@ -69,12 +69,12 @@ func (t *CallbacksTestSuite) TestFailsRunOnPanicError() {
 
 		resp := t.requestMap(r)
 		errorData := make(map[string]string)
-		errorData["ErrFrom"] = "test_error"
-		errorData["ErrMessage"] = "test error"
-
-		errorDataBytes, jsonErr := json.MarshalIndent(errorData, "", "  ")
+		jsonErr := json.Unmarshal([]byte(resp["Payload"]), &errorData)
 		t.Require().Nil(jsonErr)
-		t.Require().Equal(string(errorDataBytes), resp["Payload"])
+
+		t.Require().Equal("test_error", errorData["ErrFrom"])
+		t.Require().Equal("test error", errorData["ErrMessage"])
+		t.Require().True(errorData["StateDump"] != "")
```

Comment:
This is not the best test for now, but the ruby integration test this more thoroughly anyway.

Reply:
You sure about this? Why don't you assert some keys that we expect to exist do actually exist and have some value in them?

Reply:
Changed it to check the state dump more thoroughly.